q2_diversity/_beta/_method.py

```diff
@@ -63,6 +69,27 @@ def beta_phylogenetic(table: biom.Table, phylogeny: skbio.TreeNode,
     return results
 
 
+def beta_phylogenetic_hpc(table: BIOMV210Format, phylogeny: NewickFormat,
+                          metric: str, n_jobs: int=1,
+                          variance_adjusted: bool=False,
+                          alpha=1.0,
+                          bypass_tips: bool=False) -> skbio.DistanceMatrix:
+    if metric == 'unweighted_unifrac':
+        f = unifrac.unweighted
+    elif metric == 'weighted_unnormalized_unifrac':
+        f = unifrac.weighted_unnormalized
+    elif metric == 'weighted_normalized_unifrac':
+        f = unifrac.weighted_normalized
+    elif metric == 'generalized_unifrac':
+        f = partial(unifrac.generalized, alpha=alpha)
```

Comment:
This is fine, but a dictionary lookup might be cleaner (that's usually what we do).

Reply:
It made things cleaner, though we needed a bit of special casing though 👍